src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -421,6 +445,47 @@ public final SAMFileGATKReadWriter createSAMWriter(final File outputFile, final
     }
 
     /**
+     * Creates a VariantContextWriter who's outputFile type is determined by
+     * the outFile's extension, using the best available sequence dictionary for
+     * this tool, and default index, leniency and md5 generation settings.
+     *
+     * @param outFile output File for this writer. May not be null.
+     * @returns VariantContextWriter must be closed by the caller
```

Comment:
We have a ticket (https://github.com/broadinstitute/gatk/issues/121) for the work to create and manage these automatically.